### DISCUSSION TITLE
Flatpak not starting on new ChromeOS Flex install

### DISCUSSION THREAD

**Discussion by pa-august** [standardnotes/forum#3187] [General]
I'm attempting to run SN on a new ChromeOS Flex install and am getting:

`/app/bin/start-standardnotes: line 13: XDG_SESSION_TYPE: unbound variable`

I'm using the default Linux container which is currently still Debian 11. 
I installed the Backports version of Flatpak but that doesn't seem to have helped.

I have the SN Flatpak running fine on an official Chromebook running vanilla ChromeOS, and have had the SN Flatpak running fine on previous ChromeOS Flex installs so this behavior is new to me.

**Comment by okkei** [comment]
I have the same error message in a normal chromebook. I have not been able to use the flatpak version for several weeks. SN Version 3.165.7. Debian 11.

**Comment by grimfervor36** [comment]
https://flathub.org/setup/Chrome%20OS
Regular Repo Not Beta, Dev... Chrome OS Flex Install Penguin Terminal First Then You can get, Gnome, wine, GithubIDE or GIT from linux part, GDebi File Library for when your coding in Github But Need File! RPM Too!
was working on  Batocera in Github IDE Visual runtime for Linux installed, Github IDE CHROME BROWSER EXTENSION IN WEB STORE!
